jicofo/src/main/java/org/jitsi/jicofo/jigasi/TranscriberManager.java

```diff
@@ -125,20 +126,37 @@ private void memberPresenceChanged(@NotNull ChatRoomMember member)
         if (transcriptionStatusExtension != null
                 && TranscriptionStatusExtension.Status.OFF.equals(transcriptionStatusExtension.getStatus()))
         {
-            // puts the stopping in the single threaded executor
-            // so we can order the events and avoid indicating active = false
-            // while we are starting due to concurrent presences processed
-            executorService.execute(this::stopTranscribing);
+            active = false;
+            logger.info("detected transcription status being turned off.");
         }
         if (isRequestingTranscriber(presence) && !active)
         {
-            if (jigasiDetector == null)
+            tryToStart();
+        }
+    }
+
+    private void tryToStart()
+    {
+        if (jigasiDetector == null)
+        {
+            logger.warn("Transcription requested, but jigasiDetector is not configured.");
```

Comment:
Note for the future: it would be useful to somehow propagate this all the way to the client.

Reply:
Agreed